include/pnet_api.h

```diff
@@ -589,7 +589,8 @@ typedef int (*pnet_state_ind)(
  * value of \a idx (0x0000 - 0x7fff). All other values of \a idx are handled internally
  * by the Profinet stack.
  *
- * The application must verify the values of \a idx and \a write_length and provide a
+ * The application must verify the values of \a idx and that \a p_read_length is
```

Comment:
must verify that

Reply:
Changed to "The application must verify the value of \a idx, and that \a p_read_length is large enough."